benchmarks/README.md

```diff
@@ -24,4 +24,13 @@ passed to multiple consumers.
 - **Scatter** - Describes a fan-out one-to-many pattern whereby one function sends payloads to
 multiple recipients. Unlike the broadcast pattern, the sent payloads are different from each other.
 - **Gather** - Describes a fan-in many-to-one pattern, whereby a consumer retrieves objects from 
-multiple producers, e.g., as in the reduce phase in MapReduce.
\ No newline at end of file
+multiple producers, e.g., as in the reduce phase in MapReduce.
+
+
+## Standalone functions benchmark summary
```

Comment:
Call the ones above as "multi-function benchmarks", also please add a short intro sentence that there 2 categories of workloads in vSwarm. Check the terminology to be as in the tutorial's slides.